packages/application-shell/src/test-utils/test-utils.js

```diff
@@ -163,26 +165,114 @@ const renderWithRedux = (
     // Consumers of renderWithRedux can use
     //   { store: createReduxStore({ requestsInFlight: null, .. }) }
     // to pass an initial state to Redux.
-    store = createReduxStore(),
+    store = undefined,
```

Comment:
Now that I slept on it, maybe we should just remove `store` completely? Maybe it's the right time to tighten the API? Not sure..

Reply:
I wouldn't add a note to say that it "might be deprecated." I would only add such a note in case we are certain that we're going to get rid of it. If we need to keep it around, then it's here to stay anyways. No consumers in the MC are using it at the moment.

My gut feeling is that we can remove it - but then again I also didn't see the `sdk` mocking stuff coming up.